<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="referential">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Referential</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotation">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rotation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="apply">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_rotation">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_traslation">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>490</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-10</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="traslation">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>410</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Translation</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="radioButton_camera">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_world">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>World</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="radioButton_rotX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_rotY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_rotZ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="radioButton_transX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_transY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_transZ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="intrinsicParameter">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Intrinsic Parameter</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_f">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>430</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="f">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>430</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>f</string>
    </property>
   </widget>
   <widget class="QLabel" name="sx">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>460</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>sx</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_sx">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>460</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_sy">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>490</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="sy">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>sy</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_s0">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>430</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="s0">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>430</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>s0</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_ox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>460</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>460</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ox</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_oy">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>490</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="oy">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>490</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>oy</string>
    </property>
   </widget>
   <widget class="QLabel" name="worldParameter">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>410</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>World Parameter</string>
    </property>
   </widget>
   <widget class="QLabel" name="projectionParameter">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>410</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Projection Parameter</string>
    </property>
   </widget>
   <widget class="QLabel" name="limit">
    <property name="geometry">
     <rect>
      <x>486</x>
      <y>430</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>limit</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_limitWorld">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>430</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>8</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_elv">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>460</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>25</number>
    </property>
   </widget>
   <widget class="QLabel" name="elev">
    <property name="geometry">
     <rect>
      <x>486</x>
      <y>460</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>elev</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_azim">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>490</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>-35</number>
    </property>
   </widget>
   <widget class="QLabel" name="azim">
    <property name="geometry">
     <rect>
      <x>486</x>
      <y>490</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>azim</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_limitProjection">
    <property name="geometry">
     <rect>
      <x>634</x>
      <y>430</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="limitProction">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>430</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>limit</string>
    </property>
   </widget>
   <widget class="QLabel" name="world">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/world.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="projection">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>401</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/projection.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="camera_world">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
